tailwind.config.mjs

```diff
@@ -25,7 +25,9 @@ export default {
         'slide-in-from-right': 'slide-in-from-right 0.3s forwards ease-in-out',
       },
       boxShadow: {
-        'adder-toolbar': '0px 2px 10px 0px rgba(0, 0, 0, 0.25)',
+        // A more prominent shadow than the one used by tailwind, intended for
+        // popovers and menus
+        intense: '0px 2px 10px 0px rgb(0 0 0 / 0.25)',
```

Comment:
This is the same shadow that was previously called `adder-toolbar`. I just changed it to the newer syntax (`rgb(0 0 0 / 0.25)` instead of `rgba(0, 0, 0, 0.25)`)